controller/src/beerocks/master/beerocks_master_main.cpp

```diff
@@ -218,6 +218,33 @@ static void fill_master_config(son::db::sDbMasterConfig &master_conf,
     } else {
         master_conf.load_steer_on_vaps = std::string(load_steer_on_vaps);
     }
+
+    if (!beerocks::bpl::cfg_get_persistent_db_enable(master_conf.persistent_db)) {
+        LOG(DEBUG) << "failed to read persistent db enable, setting to default value: "
+                   << bool(DEFAULT_PERSISTENT_DB);
+        master_conf.persistent_db = (DEFAULT_PERSISTENT_DB == 1);
+    }
+    if (!beerocks::bpl::cfg_get_clients_persistent_db_max_size(
+            master_conf.clients_persistent_db_max_size)) {
+        LOG(DEBUG)
+            << "failed to read max number of clients in persistent db, setting to default value: "
+            << int(DEFAULT_CLIENTS_PERSISTENT_DB_MAX_SIZE);
```

Comment:
Since this is a macro, it will be replaced by an integer on the preprocessor so no need to cast it to int.
Apply on the other as well.

Reply:
fixed